### PR TITLE
intrinsic functions updated for cloud9

### PR DESCRIPTION
**Pull request by snjkumar23**
## What does this PR do and why?

Updated intrinsic function for cloud9 changes

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [ ] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/basics/templates/intrinsic-functions/index.md [hunk 1]
```diff
@@ -83,6 +83,26 @@ to categorize resources by purpose, owner, environment, or other criteria. Let's
...
+:::::tabs{variant="container"}
+
+::::tab{id="cloud9" label="Cloud9"}
+1. In the **Cloud9 terminal** navigate to `code/workspace`:
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+cd cfn101-workshop/code/workspace
+:::
+1. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you. Use `AmiId` which you have hardcoded in `resources.yaml` file earlier.
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+aws cloudformation update-stack --stack-name cfn-workshop-s3 --template-body file://intrinsic-functions.yaml --parameters '[{"ParameterKey":"AmiID","ParameterValue":"ami-id"}]'
```
Comment (rezabekf) [inline content/basics/templates/intrinsic-functions/index.md:96]:
```suggestion
aws cloudformation update-stack --stack-name cfn-workshop-ec2 --template-body file://intrinsic-functions.yaml --parameters '[{"ParameterKey":"AmiID","ParameterValue":"MyAmiId"}]'
```

File: content/basics/templates/intrinsic-functions/index.md [hunk 2]
```diff
@@ -83,6 +83,26 @@ to categorize resources by purpose, owner, environment, or other criteria. Let's
...
 
+
+:::::tabs{variant="container"}
+
+::::tab{id="cloud9" label="Cloud9"}
+1. In the **Cloud9 terminal** navigate to `code/workspace`:
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+cd cfn101-workshop/code/workspace
+:::
+1. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you. Use `AmiId` which you have hardcoded in `resources.yaml` file earlier.
```
Comment (rezabekf) [inline content/basics/templates/intrinsic-functions/index.md:94]:
```suggestion
1. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you. Replace the `ParameterValue` **MyAmiId** with  the value you have hardcoded in `resources.yaml` file earlier.
```

File: content/basics/templates/intrinsic-functions/index.md
Comment (rezabekf) [inline content/basics/templates/intrinsic-functions/index.md:1]: Please add C9 instructions to the challenge part, line 155-156

**Review by rezabekf — CHANGES_REQUESTED** [review]
Left few comments

File: content/basics/templates/intrinsic-functions/index.md
Comment (snjkumar23) [reply content/basics/templates/intrinsic-functions/index.md:1]: done

File: content/basics/templates/intrinsic-functions/index.md (hunk 2 above)
Comment (snjkumar23) [reply content/basics/templates/intrinsic-functions/index.md]: done

File: content/basics/templates/intrinsic-functions/index.md (hunk 1 above)
Comment (snjkumar23) [reply content/basics/templates/intrinsic-functions/index.md]: done

File: content/basics/templates/intrinsic-functions/index.md [hunk 3]
```diff
@@ -83,6 +83,26 @@ to categorize resources by purpose, owner, environment, or other criteria. Let's
...
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+cd cfn101-workshop/code/workspace
+:::
+1. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you. Replace the `ParameterValue` **MyAmiId** with the value you have hardcoded in `resources.yaml` file earlier.
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+aws cloudformation update-stack --stack-name cfn-workshop-ec2 --template-body file://intrinsic-functions.yaml --parameters ParameterKey="AmiID",ParameterValue="MyAmiId"
+:::
+1. If the `update-stack` command was successfully sent, CloudFormation will return `StackId`.
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+"StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-s3/739fafa0-e4d7-11ed-a000-12d9009553ff"
```
Comment (rezabekf) [inline content/basics/templates/intrinsic-functions/index.md:100]:
```suggestion
"StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-ec2/739fafa0-e4d7-11ed-a000-12d9009553ff"
```

**Review by rezabekf — APPROVED** [review]
LGMT

**Pull request merged**